alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md

```diff
@@ -0,0 +1,152 @@
+# ExtremelyHighIndividualControlPlaneCPU
+
+## Meaning
+
+[This alert][ExtremelyHighIndividualControlPlaneCPU](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/assets/alerts/cpu-utilization.yaml)
```

Comment:
Preview is broken here, you should change that to:
```suggestion
[This alert][ExtremelyHighIndividualControlPlaneCPU]
```
and add the following at the end of the file:
```
[ExtremelyHighIndividualControlPlaneCPU]: https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/assets/alerts/cpu-utilization.yaml
```

Same applies for `HighOverallControlPlaneCPU`.